docs/js/doc-page.js

```diff
@@ -21,6 +21,104 @@ function removeDoubleSlashes(str) {
   return str.replace(/\/{2,}/g, '/');
 }
 
+function getPathFromSlug(str) {
+  return removeDoubleSlashes(`/${str}/`);
+}
+
+function relativeToAbsUrl(url) {
+  if (url.startsWith('http') || url.startsWith('mailto')) {
```

Comment:
How could the URL argument start with 'http' or 'mailto' if it's relative?

Reply:
This is careless copy and pasting from the rest of the site where there's the possibility that a CMS-filled piece of content have both a relative or absolute URL. I'll remove it to keep it tidy, nice catch!